db/schema.rb

```diff
@@ -19,7 +19,7 @@
     t.string "record_type", null: false
     t.bigint "record_id", null: false
     t.bigint "blob_id", null: false
-    t.datetime "created_at", precision: 6, null: false
+    t.datetime "created_at", null: false
```

Comment:
I think this entire file shouldn't be committed. There's something weird going on with these precision calls (the same thing happens to me!).

Reply:
Please discard all changes to this file.